Dockerfile

```diff
@@ -54,9 +55,10 @@ ENV PYTHONUNBUFFERED=1 \
     # DEPLOY_TAG = \
     # ARTIFACTS_BUCKET = \
     # DATABRICKS_DEFAULT_IAM \
-    PORT=8000
+
```

Comment:
moved env vars around to avoid empty continuation line warnings when building dockerfiles.